### DISCUSSION TITLE
How to remove duplicates from API Response ?

### DISCUSSION THREAD

**Discussion by sirijagadeesh** [PostgREST/postgrest#2114] [Q&A] [answered]
HI all,

i am using PostgREST v.8.0.0. (docker images).
want to remove duplicates records from API response. How can i do it ?

GET /employees?select=dept_id

In SQL, we use distinct key word  and Query will become like 
select distinct dept_id from employees;

How can i write in API call

**Comment by steve-chavez** [comment, marked as answer]
Using distinct directly through the url is not supported yet: https://github.com/PostgREST/postgrest/issues/915

For now you can use a view:

```sql
create view distinct_employees as
select distinct dept_id from employees;
```

```http
GET /distinct_employees?select=dept_id
```
